config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
-  get 'market_chirp/index'
```

Comment:
We don't need this as setting the root to this route automatically handles get for us, plus we don't want to get 'market_chirp/index'.